architecture/additional_concepts/storage.adoc

```diff
@@ -596,7 +596,7 @@ metadata:
 spec:
   containers:
     - name: myfrontend
-      image: dockerfile/nginx
+      image: dockerfile/redis
```

Comment:
let's not change this one.  it doesn't run with `dockerfile/nginx` anyway and the volumeMounts below don't make sense with redis.